builds/e2e/runner.sh

```diff
@@ -21,5 +22,11 @@ EOF
 mkdir -p /etc/systemd/system/docker.service.d/
 cp ~/proxy-env.override.conf /etc/systemd/system/docker.service.d/
 
+# Make proxy-env.override.conf available in $user's home directory so tests can
+# apply the same proxy settings to the iotedge service
+home=$(eval echo ~$user)
```

Comment:
Quotes around the `$()` please. `home="$(...)"`

Reply:
And quotes around the uses of `$home/` and `$user:$user` etc below too.

Reply:
Done.